swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -10863,14 +11127,31 @@ components:
       description: |
         If it equals "true", the TPP prefers a redirect over an embedded SCA approach.
         If it equals "false", the TPP prefers not to be redirected for SCA. The ASPSP will then choose between the Embedded or the Decoupled
-        SCA approach, depending on the choice of the SCA procedure by the TPP/PSU.
+        SCA approach, depending on the parameter TPP-Decoupled-Preferred and the choice of the SCA procedure by the TPP/PSU.
         If the parameter is not used, the ASPSP will choose the SCA approach to be applied depending on the SCA method chosen by the
         TPP/PSU.
       schema:
         type: boolean
       required: false
 
 
+    TPP-Decoupled-Preferred:
+      name: TPP-Decoupled-Preferred
+      in: header
+      description: |
+        If it equals "true", the TPP prefers a decoupled SCA approach. 
+        If it equals "false", the TPP prefers not to use the decoupled approach for SCA. The ASPSP will then choose between the embedded or the redirect SCA approach, depending on the choice of the SCA procedure by the TPP/PSU. 
+        If the parameter is not used, the ASPSP will choose the SCA approach to be applied depending on the parameter TPP-Redirect-Preferred and the SCA method chosen by the TPP/PSU. 
+        The parameter might be ignored by the ASPSP.
+        If both parameters TPP-Redirect-Preferred and TPP-Decoupled-Preferred are present and true, the request is still not rejected, but it is up to the ASPSP, which approach will actually be used.
+        
+        **Remark for Future:** 
+        TPP-Redirect-Preferred and TPP-Decoupled-Preferred will be revised in future versions, maybe merged. Currently kept separate for downward compatibility.
+      schema:
+        type: boolean
+      required: false
+      
+      
```

Comment:
Relevant Change: TPP-Decoupled-Preferred is new
Adoption approach: Follow the change